docs/4-benchpark-setup.rst

```diff
@@ -10,7 +10,7 @@ Also choose a directory for your experiment::
 where:
 
 - ``benchmark/ProgrammingModel``: amg2023/openmp | amg2023/cuda | saxpy/openmp (available choices in ``benchpark/experiments``)
-- ``system``: ats2 | ats4 | cts1 (available choices in ``benchpark/configs``)
+- ``system``: x86_64 | HPECray-EPYC3-MI250X-Slingshot | AWS-Hpc7a-EPYC4-EFA (available choices in ``benchpark/configs``)
```

Comment:
This needs to update to match the other changes.

Reply:
yeah, some of these need to be automated